src/Sidebar.js

```diff
@@ -20,29 +20,20 @@ import "./Sidebar.css"
 
 function Sidebar() {
   const [channels, setChannels] = useState([])
-  // const roomsCol = collection( db, "rooms");
-  // const snapShot = getDocs(roomsCol);
-  console.log(db)
+  useEffect(()=>{
+    async function fetchChannels() {
```

Comment:
This was bug 2: Async code must be wrapped in an async function when doing it inside a `useEffect`

Reply:
Yes, it was a question I had. It is clear now. I am confident I would not make mistake with the async/awaut function again.